tests/wpt/meta-legacy-layout/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/sticky-content.html.ini

```diff
@@ -0,0 +1,4 @@
+tests/wpt/meta-legacy-layout/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/sticky-content.html.ini
```

Comment:
I think we have a syntax error - the path here needs to be wrapped in `[` `]` to form a valid key. This is the reason the latest try run has failed.

Reply:
Whoops. I've fixed this. Thanks.